.circleci/config.yml

```diff
@@ -109,6 +109,9 @@ jobs:
         type: string
       certora_cli_package:
         type: string
+      examples_repo_path:
```

Comment:
That doesn't really hurt but it's also not needed since that path can't be anything else (correct me if I'm wrong). So I would prefer that you delete this parameter.

Reply:
Make testing so much easier.

Reply:
It can be a specific test instead of running the entire folder every time.

Reply:
Ohh I see. Yes good point.